### PR TITLE
#159082370 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by lytes20**
#### What does this PR do?
- Integrate TravisCI with readme badge plus coveralls and code climate maintainability badge
#### Description of Task to be completed?
- This is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product
#### How should this be manually tested? 
- The tests are run using the command `python3 manage.py test --settings=authors.settings.test`
- This chore is executed everytime  push is made to the repository and PR is created
#### Any background context you want to provide?
- Badges pointing to both develop and this branch have been included. We need a successful build on develop branch to activate the coveralls badge
#### What are the relevant pivotal tracker stories?
- [#159082370](https://www.pivotaltracker.com/story/show/159082370)
#### Screenshots
screen shot 2018-07-27 at 14 32 30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+# Develop Branch badges
 [![Build Status](https://travis-ci.org/andela/ah-backend-spaces.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-spaces)
 [![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-spaces/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-spaces?branch=develop)
 [![Maintainability](https://api.codeclimate.com/v1/badges/96588e5cfb3ba7da70bb/maintainability)](https://codeclimate.com/github/andela/ah-backend-spaces/maintainability)
 
+# ch-travisci-badge-159082370 branch badges
```
Comment (ssewilliam) [inline README.md]: Why do we have the badges here too

I believe it should be 'develop branch's badges only

**Comment by abulojoshua1** [comment]
Looks good.

**Review by ssewilliam — COMMENTED** [review]
Why do we have the badges here too

I believe it should be 'develop branch's badges only

File: README.md (hunk 1 above)
Comment (geofrocker) [reply README.md]: Just for testing. The should be removed later

File: requirements.txt [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+coveralls==1.3.0
 Django==2.0.6
 django-cors-middleware==1.3.1
 django-coverage==1.2.4
 django-extensions==2.1.0
 django-nose==1.4.5
 djangorestframework==3.8.2
 djangorestframework-jwt==1.11.0
+docopt==0.6.2
+idna==2.7
```
Comment (geofrocker) [inline requirements.txt:15]: why did you remove psycopg2-binary yet it is required

File: README.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+
 [![Build Status](https://travis-ci.org/andela/ah-backend-spaces.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-spaces)
-[![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-spaces/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-spaces?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-spaces/badge.svg?branch=ch-travisci-badge-159082370)](https://coveralls.io/github/andela/ah-backend-spaces?branch=develop)
```
Comment (geofrocker) [inline README.md]: Did you forget to update this?

File: requirements.txt (hunk 2 above)
Comment (lytes20) [reply requirements.txt:15]: My bad, I just did a `pip freeze > requirements.txt`. I guess it was not in the requirements but let me rectify this

File: README.md (hunk 3 above)
Comment (lytes20) [reply README.md]: I updated this

Comment (geofrocker) [reply README.md]: It is still pointing to your branch
